pages/objects/[id].vue

```diff
@@ -119,16 +125,24 @@ async function downloadResource(endpointId?: string) {
   }
 }
 
+function find_parent(relations: v2Relation[]): string | undefined {
+  for (const relation of relations) {
+    if (!relation.internal) {
+      continue;
+    }
+    if (relation.internal.definedVariant !== v2InternalRelationVariant.INTERNAL_RELATION_VARIANT_BELONGS_TO) {
+      continue;
+    }
+    if (relation.internal.direction !== v2RelationDirection.RELATION_DIRECTION_INBOUND) {
+      continue;
+    }
+    return relation.internal.resourceId;
+  }
+  return null;
+}
```

Comment:
This function takes the first parent found in the resource's relationships. However, it is possible for a resource to have multiple parents, but it is not guaranteed that the user has the same permissions for all of them. 

This could result in the id of a parent being passed to the resource creation form for which the user has no permissions, ultimately causing the resource creation to fail. Although it is not possible to definitively determine which parent the user wants to have in the case of multiple parents, it should at least be checked that the user is given a parent id under which they can create sub-resources.